.github/workflows/stale.yml

```diff
@@ -0,0 +1,18 @@
+name: "Close stale issues"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been open many days with no activity. It will be closed soon unless the stale label is removed or a comment is made.'
```

Comment:
```suggestion
        stale-issue-message: 'This issue is stale because it has been waiting on the original poster to reply for many days with no activity. It will be closed soon unless the stale label is removed or a comment is made.'
```